vulns/CVE-2024-57841.yml

```diff
@@ -0,0 +1,7 @@
+reachability: Local
```

Comment:
I'm wondering if the reachability might actually be Remote in this case, as the affected API is tcp_conn_request().

Reply:
Yes, I agree :)